code/modules/martial_arts/combos/judo/discombobulate.dm

```diff
@@ -0,0 +1,16 @@
+/datum/martial_combo/judo/discombobulate
+	name = "Discombobulate"
+	steps = list(MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB)
+	explaination_text = "Deliver a palm strike to your opponents ear, briefly confusing them"
+	combo_text_override = "Disarm, Grab"
+
+/datum/martial_combo/judo/discombobulate/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(user.IsStunned())
+		return MARTIAL_COMBO_FAIL
```

Comment:
this check isnt needed, if they were stunned they wouldnt be able to click on them in the first place to perform the combo